extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js

```diff
@@ -96,45 +96,66 @@ RdfSchemaAlignmentDialog.prototype._constructBody = function(body) {
       }
     }, function(rdfSkeleton) {
       $.getJSON("command/core/get-operations?" + $.param({
-        project: theProject.id
-      }), function(json) {
-        var operations_old = json.entries.map(function(val) {
-          return val['operation'];
-        });
-        
-        if (operations_old.length == 1) {
-          operations_old = [];
-        } else {
-          operations_old = operations_old.slice(-5);
-        }
-        var operations_new = JSON.parse(rdfSkeleton['skeleton']).operations;
-        Refine.postCoreProcess("apply-operations", {}, {
-          operations: JSON.stringify(operations_old.concat(operations_new))
-        }, {
-          everythingChanged: true
-        }, {
-          onDone: function(o) {
-            if (o.code == "pending") {
-              Refine.update({
-                everythingChanged: true
-              });
-            }
+          project: theProject.id
+        }), function(json) {
+          var operations_old = json.entries.map(function(val) {
+            return val['operation'];
+          });
+
+          if (operations_old.length == 1) {
+            operations_old = [];
+          } else {
+            operations_old = operations_old.slice(-5);
           }
+          var operations_new = JSON.parse(rdfSkeleton['skeleton']).operations;
+          Refine.postCoreProcess("apply-operations", {}, {
+            operations: JSON.stringify(operations_old.concat(operations_new))
+          }, {
+            everythingChanged: true
+          }, {
+            onDone: function(o) {
+              if (o.code == "pending") {
+                Refine.update({
+                  everythingChanged: true
+                });
+              }
+            }
+          });
         });
-      });
+      var prefixes = [];
+      var new_prefixes = JSON.parse(rdfSkeleton['skeleton']).prefixes;
+      if (self._schema.prefixes != undefined) {
+        prefixes = prefixes.concat(self._schema.prefixes).concat(new_prefixes);
+      } else if (self._schema.prefixes.length == 0) {
+        prefixes = self._schema.prefixes;
+      } else {
+        prefixes = prefixes.concat(new_prefixes);
+      }
```

Comment:
changes are above starting at line 125

Reply:
I think @rajaram5 was referring to the prefixes from the json blob.

Reply:
@Shamanou I meant this variable `new_prefixes` check that this variable is not undefined or empty